WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt

```diff
@@ -915,6 +878,101 @@ secret_taoism_community = {
 	}
 }
 
+<<<<<<< HEAD
```

Comment:
Are you joking with me? I linked you to this yesterday, and reminded you of it before the old pull was closed https://github.com/loup99/WtWSMS/commit/0adebcde62546134b1bb2faad62016ca2fdd8054#diff-b7f0f652709d2e453cc5b687c40fccedR881